services/ui-src/src/utils/other/parsing.ts

```diff
@@ -51,3 +50,128 @@ export const sanitizeAndParseHtml = (html: string) => {
   const parsedHtml = parse(sanitizedHtml);
   return parsedHtml;
 };
+
+// parsing the field name numbers for the PDF preview page
+export const parseFieldLabel = (labelObject: {
+  label?: string;
+  hint?: string;
+  hideHint?: boolean;
+}) => {
+  const labelArray = labelObject.label?.split(" ");
+  if (labelArray) {
+    const indicator = labelArray[0];
+    labelArray.shift();
+    const label = `<p><strong>${labelArray.join(" ")}</strong></p>${
+      labelObject.hint && !labelObject.hideHint
+        ? `<p>${labelObject.hint}</p>`
+        : ""
+    }`;
+    return {
+      indicator,
+      label,
+    };
+  }
+  return {
+    indicator: "",
+    label: "",
+  };
+};
+
+const noResponse = `<p style="color:#9F142B">Not Answered</p>`;
```

Comment:
Post-merge note: Should refactor to just return the text and handle the styling separately.